evennia/utils/evmenu.py

```diff
@@ -1126,17 +1226,174 @@ def node_formatter(self, nodetext, optionstext):
         """
         sep = self.node_border_char
 
+        if type(nodetext) == dict:
+            if 'text' in nodetext:
+                text = nodetext['text']
+            else:
+                text = ''
+            if 'footer' in nodetext:
+                footer = nodetext['footer']
+            else:
+                footer = False
+        else:
+            text = str(nodetext)
+            footer = False
+        if type(nodetext) == dict and 'format' in nodetext:
+            if nodetext['format'] == 'scroll':
+                result = self.scroll(text, optionstext, footer)
+            else:
+                result = self.default_format(sep, text, optionstext, footer)
+        else:
+            result = self.default_format(sep, text, optionstext, footer)
+
+        return result
+
+    def default_format(self, sep, text, optionstext, footer):
+        """
+        Original EvMenu formatting style.
+
+        sep (str): Seperator character
+        text (str): The node text as originally returned by `self.nodetext_formatter`.
+        optionstext (str): The options display as originally returned by `self.options_formatter`.
+        footer (str or none): An footer applied after the options.
+
+        Returns:
+            node (str): The formatted node to display.
+
+        """
         if self._session:
             screen_width = self._session.protocol_flags.get("SCREENWIDTH", {0: _MAX_TEXT_WIDTH})[0]
         else:
             screen_width = _MAX_TEXT_WIDTH
 
-        nodetext_width_max = max(m_len(line) for line in nodetext.split("\n"))
+        nodetext_width_max = max(m_len(line) for line in text.split("\n"))
         options_width_max = max(m_len(line) for line in optionstext.split("\n"))
         total_width = min(screen_width, max(options_width_max, nodetext_width_max))
         separator1 = sep * total_width + "\n\n" if nodetext_width_max else ""
         separator2 = "\n" + sep * total_width + "\n\n" if total_width else ""
-        return separator1 + "|n" + nodetext + "|n" + separator2 + "|n" + optionstext
+        result = separator1 + "|n" + text + "|n" + separator2 + "|n" + optionstext
+        if footer:
+            result = result + '|/' + footer
+        return result
+
+    def scroll(self, text, options, footer):
```

Comment:
As cool as the effect looks, this does not belong in the core EvMenu. It may be a good contrib though, for people to import and use as a formatter for a specific node.

Reply:
I wanted to try to include something that had a more visible effect on the layout of a node to illustrate that there's a great deal of ability in the simple ``format`` (or whatever we eventually name it) function. However, I can probably do that in the tutorials and we can replace this with something simpler such as ``no_bars`` or ``invert`` (placing menu options above the text).

Given the simplicity of those routines we could probably even add both unless you are worried that too many alternate layout methods will bloat the routine.